daemon/mgr/container_state.go

```diff
@@ -90,6 +90,7 @@ func (c *Container) SetStatusRunning(pid int64) {
 	c.State.StartedAt = time.Now().UTC().Format(utils.TimeLayout)
 	c.State.Pid = pid
 	c.State.ExitCode = 0
+	c.State.Running = true
```

Comment:
Do we need to set `c.State.Running` to flase when container stop?

Reply:
Good point, i will optimize soon